error_test.go

```diff
@@ -43,7 +43,7 @@ func (err *errTest) Error() string {
 
 // Format implements fmt.Formatter.
 func (err *errTest) Format(f fmt.State, c rune) {
-	if !f.Flag('+') || c != 'v' {
+	if c != 'v' {
```

Comment:
Why has this changed, out of interest?

Reply:
Using the `%w` verb with `fmt.Errorf` to wrap an error causes the value to be formatted with the `%v` verb, so the test output kept including the "unexpected verb for formatting the error" warning. I thought about either including that in the test output or introducing a new error type or value, but this seemed more straight forward.

Happy to change the tests to use a different error type or value if we'd like to keep this here?